paddle/fluid/operators/distributed/send_recv.proto.in

```diff
@@ -80,6 +80,7 @@ message VariableMessage {
   // when profile switches from 1 to 2.
   int64 profile = 11;
   int64 trainer_id = 12;
+  string table_name = 13;
```

Comment:
is it possible to add a `VarType` than attributes? for prefetch, it should be a `vector<std:string>` of length 3.

Reply:
will optimize in the future pr.